{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "11a0393b_5060619a",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 2
      },
      "lineNbr": 0,
      "author": {
        "id": 1018946
      },
      "writtenOn": "2023-01-12T12:55:14Z",
      "side": 1,
      "message": "Tested downstream and with the fix image-sets import is back working.",
      "revId": "a8edea28f6e44c9f230f5e59495fabf0ca6b0e89",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735"
    }
  ]
}